help2man/help2man.inc

```diff
@@ -0,0 +1,9 @@
+# -*- mode:python; -*-
+RECIPE_TYPES = "machine native"
+
+EXTRA_AUTORECONF="-Iaclocal.m4"
+inherit autotools-autoreconf
```

Comment:
can't tell i have to test in on my fedora, centos 6 and ubuntu in order to see if the tools versions are good enough